src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Formatting/HtmlFormattingPass.cs

```diff
@@ -224,8 +224,7 @@ private static List<TextChange> AdjustRazorIndentation(FormattingContext context
     private static bool IsPartOfHtmlTag(FormattingContext context, int position)
     {
         var syntaxTree = context.CodeDocument.GetSyntaxTree();
-        var change = new SourceChange(position, 0, string.Empty);
-        var owner = syntaxTree.Root.LocateOwner(change);
+        var owner = syntaxTree.Root.FindInnermostNode(position, includeWhitespace: true);
```

Comment:
Include whitespace may not be needed here...

Reply:
If the tests all pass with it as `false` then no objection here, but I didn't check